### PR TITLE
add Vendor dir to repository

### PR DESCRIPTION
**Pull request by delgod**

### CODE REVIEW

**Comment by AlekSi** [comment]
> Beside that there is an issue with diff noise

It should be hidden by default on Github.

File: .travis.yml [hunk 1]
```diff
@@ -14,9 +14,12 @@ notifications:
 
 install:
   - go get -u github.com/Masterminds/glide
+  - rm -rf vendor/
```
Comment (AlekSi) [inline .travis.yml]: That's… interesting, to say at least. What's the point of vendoring then?

Comment (arvenil) [inline .travis.yml]: It's a test on `.travis` to be sure we didn't sneak any dev changes to vendor dir.
Take a look down below:
```
- git diff --exit-code
```
https://github.com/percona/pmm-client/pull/27/files/149e76875da5d49119b3f675f45f548e632d2b89#diff-354f30a63fb0907d4ad57269548329e3R21

We can add later better comments.

Comment (arvenil) [inline .travis.yml]: We grab source, we remove vendor dir, we refetch dependencies according to lock file, we check if vendor dir is equal to fetched dependencies.

Comment (AlekSi) [inline .travis.yml]: That's ingenious. I like it.

**Pull request merged**